addons/mass_mailing/views/utm_campaign_views.xml

```diff
@@ -29,6 +29,7 @@
                             <button name="action_duplicate" type="object" string="Duplicate"/>
                         </tree>
                     </field>
+                    <field name="is_campaign_activated" invisible="1"/>
```

Comment:
Would move it above first usage in header override, just to keep things a bit ordered